perun-openapi/openapi.yml

```diff
@@ -2895,6 +2922,19 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/utils/getAppsConfig:
+    get:
+      tags:
+        - Utils
+      operationId: getAppsConfig
+      summary: Gets configuration of perun apps brandings and apps' domains
+      description: Returns object of type PerunAppsConfig
+      responses:
+        '200':
+          $ref: '#/components/responses/StringResponse'
```

Comment:
I think this should be `PerunAppsConfigResponse`.

Reply:
Right, I think I forgot to commit this change. Fixed.